django_keycloak/managers.py

```diff
@@ -93,31 +34,22 @@ def create_from_token(self, token, password=None, **kwargs):
     def get_by_keycloak_id(self, keycloak_id):
         return self.get(id=keycloak_id)
 
-    def create_keycloak_user(self, *args, **kwargs):
-        keycloak_user = self._create_user_on_keycloak(*args, **kwargs)
-        return self.create(
-            id=keycloak_user.get("id"),
-            username=keycloak_user.get("username"),
-        )
-
 
 class KeycloakUserManagerAutoId(KeycloakUserManager):
-    def create_from_token(self, token, password=None, **kwargs):
+    def create_from_token(self, token: Token, **kwargs):
         """
-        Create a new user from a valid token
+        Create a local new user from a valid token
         """
-        if not self.keycloak.is_token_active(token):
-            raise ValueError("Invalid token")
 
-        user_info = self.keycloak.get_user_info(token)
+        user_info = token.user_info
 
         # set admin permissions if user is admin
-        is_staff = False
-        is_superuser = False
-        if self.keycloak.has_superuser_perm(token):
-            is_staff = True
-            is_superuser = True
+        if token.superuser:
```

Comment:
Function name should be `has_superuser_perm` or possibly be renamed to `is_superuser` here and in Token

Reply:
is_superuser would match the naming used by Django's auth module. They use it for their PermissionMixin, AbstractBaseUser and AnonymousUser, so it would match the style. There it is a boolean field, so using a property method would match that style

Reply:
Alright, `is_superuser` property then, let's keep the consistency